analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/TokenTypeAnalyzerTest.Classifier.cs

```diff
@@ -958,9 +958,10 @@ public void M()
     [DataRow("M([u:MethodGroup]<int>);", false)]
     [DataRow("M([u:MethodGroup]<T>);", false)]
     [DataRow("M(C<T>.[u:StaticMethodGroup]<T>);", false)]
+    [DataRow("global::[u:System].Diagnostics.Debug.WriteLine(\"Message\");", true)]
+    [DataRow("global::System.Diagnostics.[t:Debug].WriteLine(\"Message\");", true)]
```

Comment:
There were 2 problems:
- System was miss classified as type although it should be unknown
- Debug was classified as unknown although it should be classified as type